Assets/Scripts/Models/Furniture.cs

```diff
@@ -791,8 +794,8 @@ private Furniture Clone()
     private bool DefaultIsValidPosition(Tile t)
```

Comment:
The scope of this PR is primarily to get rid of build warnings, StyleCop warnings and at least some of the MonoDevelop auto-corrections. I'm fixing this since I've already touched most lines in this function.